### PR TITLE
[E2E] Formatters

### PR DESCRIPTION
**Pull request by vlad-dargel**

### CODE REVIEW

File: tests/e2e/test-data/formatters-data.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+export const formatters = [{
```
Comment (ofersteinberg) [inline tests/e2e/test-data/formatters-data.ts:1]: Please add on top a short description of the file's purpose.

File: tests/e2e/tests/critical-path/browser/formatters.e2e.ts [hunk 2]
```diff
@@ -51,137 +36,166 @@ fixture `Formatters`
...
-            // Verify that json value is formatted and highlighted
-            await t.expect(valueSelector.find(browserPage.cssJsonValue).exists).ok(`${keysData[i].textType} Value is not formatted to JSON`);
-            // Verify that Hash field is formatted to json and highlighted
-            if (keysData[i].keyName === 'hash') {
-                await t.expect(browserPage.hashField.find(browserPage.cssJsonValue).exists).ok('Hash field is not formatted to JSON');
-            }
-            // Verify that Stream field is formatted to json and highlighted
-            if (keysData[i].keyName === 'stream') {
-                await t.expect(Selector(browserPage.cssJsonValue).count).eql(2, 'Hash field is not formatted to JSON');
+formattersHighlightedSet.forEach(formatter => {
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/browser/formatters.e2e.ts:39]: Does it work well?

File: tests/e2e/test-data/formatters-data.ts (hunk 1 above)
Comment (vlad-dargel) [reply tests/e2e/test-data/formatters-data.ts:4]: thanks, updated

File: tests/e2e/tests/critical-path/browser/formatters.e2e.ts (hunk 2 above)
Comment (vlad-dargel) [reply tests/e2e/tests/critical-path/browser/formatters.e2e.ts:39]: I've used this from testcafe documentation https://testcafe.io/documentation/402804/recipes/best-practices/create-data-driven-tests
and it works perfect

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**